README.md

```diff
@@ -10,24 +10,40 @@ This is the code behind: https://www.digitale-gesellschaft.ch/auskunftsbegehren
 Please open issues here for anything you find not working correctly with the Application. If you miss any organisation or company in the data, feel free to open Pull Requests at the [Data Repository](https://github.com/DigitaleGesellschaft/Datenauskunftsbegehren-Data).
 
 ## Develop
-### Setup local development environment
-*Note that you will need to have [Node.js](https://nodejs.org) installed.*
+
+### Prerequisites
+
+* Install [Node.js LTS](https://nodejs.org)
+* Create self signed SSL Certificate for localhost, required for `npm run dev`. Recommended CLI: [mkcert](https://github.com/FiloSottile/mkcert)
+
+```bash
+mkcert -install && \
+mkcert localhost 127.0.0.1 && \ 
+mv localhost+1-key.pem local.key && \ 
+mv localhost+1.pem local.cert
+```
+
+* Make sample data locally available: `cp ./public/data.example.json ./public/data.json` 
```

Comment:
I will document the workflow to use the actual live data from https://github.com/DigitaleGesellschaft/Datenauskunftsbegehren-Data instead.

Reply:
Ok, maybe crosslinking these two repos in the readme would help other contributor ;)

Reply:
Only one way linked yet: https://github.com/DigitaleGesellschaft/Datenauskunftsbegehren#contribute
I absolutely agree that better contribution/setup/development guides would be nice. There was really not too much time between launch and vacation. Happy to take PRs regarding this. If unsure how things could/should be done, don't hesitate to open Issues/use chat to discuss first.